src/coreComponents/common/format/table/TableFormatter.cpp

```diff
@@ -79,13 +82,47 @@ string TableCSVFormatter::toString< TableData >( TableData const & tableData ) c
 ///////////////////////////////////////////////////////////////////////
 
 /**
- * @brief Build  cell given an alignment, a value and spaces
+ * @brief distributes spaces equally from a string vector
+ * @param vec The vector where we adding spaces
+ * @param totalSpaces Total spaces to be distributed
```

Comment:
I have issues for really understand the point of this method, would it be a good rewrite?
```suggestion
 * @param totalSpaces Total spaces to distribute over all strings
```
If not, please fix it so it is as much accurate as possible.

Reply:
the new description fit better for understanding